### PR TITLE
Add ARW enum and remove ARW constants

### PR DESCRIPTION
**Pull request by gpetrou**
## Proposed changes

- Add ARW enum.
- Remove ARW constants and replace their usages with the above enum.

### CODE REVIEW

File: src/Common/src/Interop/User32/Interop.ARWDirection.cs [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+internal static partial class Interop
+{
+    internal static partial class User32
+    {
+        public enum ARWDirection : long
```
Comment (hughbe) [inline src/Common/src/Interop/User32/Interop.ARWDirection.cs]: please name this `ARW`

File: src/Common/src/Interop/User32/Interop.ARWStartingPosition.cs [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+internal static partial class Interop
+{
+    internal static partial class User32
+    {
+        public enum ARWStartingPosition : long
```
Comment (hughbe) [inline src/Common/src/Interop/User32/Interop.ARWStartingPosition.cs]: just make this part of the enum above

File: src/Common/src/Interop/User32/Interop.ARWDirection.cs (hunk 1 above)
Comment (gpetrou) [reply src/Common/src/Interop/User32/Interop.ARWDirection.cs]: Done.

File: src/Common/src/Interop/User32/Interop.ARWStartingPosition.cs (hunk 2 above)
Comment (gpetrou) [reply src/Common/src/Interop/User32/Interop.ARWStartingPosition.cs]: Sure.

File: src/Common/src/Interop/User32/Interop.ARW.cs [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+internal static partial class Interop
+{
+    internal static partial class User32
+    {
+        public enum ARW : long
```
Comment (RussKie) [inline src/Common/src/Interop/User32/Interop.ARW.cs]: Could you please explain why it is `long`?
C++ `long` is 32 bits, the same as C# `int`; C# `long` is 64 bits which is C++ `long long`.

@JeremyKuhne could you please confirm

Comment (JeremyKuhne) [reply src/Common/src/Interop/User32/Interop.ARW.cs]: As @RussKie pointed out, `long` in C/C++ on Windows is always 32 bits, so this should remain `int`.

Comment (gpetrou) [reply src/Common/src/Interop/User32/Interop.ARW.cs]: Fixed.

**Comment by RussKie** [comment]
@hughbe  any objections?

**Comment by hughbe** [comment]
Nope looks good

**Pull request merged**